install-non-oss.sh

```diff
@@ -54,3 +54,5 @@ mvn install:install-file -Dfile=vapi-authentication-2.37.0.jar -DgroupId=com.vmw
 mvn install:install-file -Dfile=vsphereautomation-client-sdk-3.3.0.jar -DgroupId=com.vmware.vsphereautomation.client -DartifactId=vsphereautomation-client-sdk -Dversion=3.3.0 -Dpackaging=jar
 mvn install:install-file -Dfile=vsphereautomation-client-sdk-4.0.0.jar -DgroupId=com.vmware.vsphereautomation.client -DartifactId=vsphereautomation-client-sdk -Dversion=4.0.0 -Dpackaging=jar
 
+# From https://libvirt.org/maven2/net/juniper/tungsten/juniper-tungsten-api/2.0/
```

Comment:
URL returns 404? (also ssl/https errors)

Reply:
yes, @rohityadavcloud builds fail now, we copied from repo to make it work in trillian again. Radu's repo doesn't exist anymore either.

Reply:
ah that's not good @DaanHoogland - do we know if this jar is published somewhere or the source repo of that?